### PR TITLE
Readme file created

### PR DESCRIPTION
**Pull request by suraj-rathod**

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -0,0 +1,64 @@
+=== Knawat WooCommerce DropShipping WPML Addon ===
+Contributors: knawat,surajrathod,dharm1025
+Tags: dropshipping, woocommerce, woocommerce dropshipping, dropship, woo dropshipping, knawat, knawat dropshipping, multivendor marketplace, WCFM Marketplace, Multi seller, Marketplace, WPML Addon
+Requires at least: 1.0.0
+Tested up to: 4.8.0
```
Comment (iamdharmesh) [inline readme.txt:5]: add WP's latest version here.

File: readme.txt [hunk 2]
```diff
@@ -0,0 +1,64 @@
+=== Knawat WooCommerce DropShipping WPML Addon ===
+Contributors: knawat,surajrathod,dharm1025
+Tags: dropshipping, woocommerce, woocommerce dropshipping, dropship, woo dropshipping, knawat, knawat dropshipping, multivendor marketplace, WCFM Marketplace, Multi seller, Marketplace, WPML Addon
```
Comment (iamdharmesh) [inline readme.txt:3]: They show first 5 tags only. So, add WPML in first five 5 tags.

File: readme.txt [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+1. Download the "Knawat WooCommerce DropShipping WPML Addon" plugin from WordPress.org.
+2. Unzip the package and move to your plugins directory.
+3. Log into WordPress and navigate to the "Plugins" screen.
+4. Locate "Knawat WooCommerce DropShipping WPML Addon" in the list and click the "Activate" link.
+
+
+== Screenshots ==
+
+1. 
+2.
```
Comment (iamdharmesh) [inline readme.txt:49]: Add screenshots please. in not needed then remove this section please.

File: readme.txt [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+== Description ==
+
+===Knawat WooCommerce DropShipping WPML Addon===
+
+Knawat WooCommerce DropShipping WPML Addon
+
+
+== Features ==
+
+WooCommerce DropShipping WPML Addon Translator for WooCommerce Products, Categories, Tags
```
Comment (iamdharmesh) [inline readme.txt:20]: WooCommerce DropShipping WPML Addon Adds WPML support to Knawat WooCommerce DropShipping plugin, which enable translation of Knawat products, categories and tags using WPML.

This addon is comes with zero configuration you just need to install and activate. rest things will be handled automatically.

**Review by iamdharmesh — CHANGES_REQUESTED** [review]

File: readme.txt (hunk 4 above)
Comment (suraj-rathod) [reply readme.txt:20]: Thanks for the details @iamdharmesh I will update all points and let you know

File: readme.txt (hunk 1 above)
Comment (suraj-rathod) [reply readme.txt]: @iamdharmesh please review latest commit i have update it

**Review by iamdharmesh — APPROVED** [review]

**Pull request merged**